docs/model.md

```diff
@@ -0,0 +1,122 @@
+# Heapster Metric Model
+
+The Heapster Model is a structured representation of metrics for Kubernetes clusters, which is exposed through a set of REST API endpoints.
+It allows the extraction of up to 1 hour of historical data for any Container, Pod, Node or Namespace in the cluster, as well as the cluster itself.
```

Comment:
not for historical data, meaning that you can't extract a timeseries for more than 1 hour ago.
As we discussed offline, the DayStore only provides derived stats using the hourly derived stats